Sources/Kitura/Router.swift

```diff
@@ -102,7 +118,12 @@ public class Router {
     /// ```
     /// - Parameter mergeParameters: Optional parameter to specify if the router should be able to access parameters
     ///                                 from its parent router. Defaults to `false` if not specified.
+    /// Initialize a `Router` instance
```

Comment:
These comments seem to be duplicated from above, please remove.

Reply:
done